README.md

```diff
@@ -23,6 +23,10 @@ Feel free to add:
 - [pytest](base/pytest.md)
 - [python](base/python.md)
 
+### [Code](code/README.md)
+
+- [imports](code/imports.md)
+
```

Comment:
I would have add this to the precedent section, Admin, which have already documentation about poetry, python,… Of course, the name Admin could also be changed to be easier to understand.